.github/workflows/tests.yml

```
@@ -24,7 +24,15 @@ jobs:
         run: bun install
       - name: Run tests
-        run: bun test
+        run: bun test tests
+        env:
+          ENCRYPTION_KEY: ${{ secrets.ENCRYPTION_KEY }}
+          SERVICE_ACCOUNT_ENCRYPTION_KEY: ${{ secrets.SERVICE_ACCOUNT_ENCRYPTION_KEY }}
+      - name: Install Dependencies (Frontend)
+        working-directory: ./frontend
+        run: bun install
+      - name: Run tests
+        run: bun run test 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)